tools/buildgen/generate_build_additions.sh

```diff
@@ -31,7 +31,7 @@ gen_build_yaml_dirs="  \
 gen_build_files=""
 for gen_build_yaml in $gen_build_yaml_dirs
 do
-  output_file=`mktemp /tmp/genXXXXXX`
+  output_file=$(mktemp /tmp/gen_$(echo $gen_build_yaml | tr '/' '_').yaml.XXXXX)
```

Comment:
nit: why this change?

Reply:
We have 12 gen_build_yaml script in our repo, this will help us identify which intermediate file is generated by which script. E.g., `src/re2/gen_build_yaml.py` generates `gen_src_re2_gen_build_yaml_py.yaml.12345` instead of `gen12345`.